app/src/components/misc/SupportPanel/sendSlackInvite.ts

```diff
@@ -0,0 +1,36 @@
+import { message } from "antd";
+import { getFunctions, httpsCallable } from "firebase/functions";
+
+interface ResInterface {
+  success: boolean;
+  message: string;
+  data: object;
+}
+
+export default async function sendSlackInvite() {
+  const hide = message.loading({
+    content: "Sending Slack Connect Invitation",
+    className: "slack-connect-toast",
+    duration: 0,
+  });
+  const sendSlackInviation = httpsCallable<null, ResInterface>(getFunctions(), "slackConnect-sendSlackInvitation");
```

Comment:
I don't think typecasting the response for this function is going to be useful in any way. we can ignore that